install/kubernetes/cilium/values.yaml

```diff
@@ -1016,6 +1016,8 @@ socketLB:
   enabled: false
   # -- Disable socket lb for non-root ns. This is used to enable Istio routing rules.
   # hostNamespaceOnly: false
+  # -- Enable terminating pod connections to deleted service backends.
+  # terminatePodConnections: true
```

Comment:
Commenting this out will mean it's undocumented in the Helm reference (due to a bug in helm-docs, c.f. https://github.com/cilium/cilium/pull/20639#issuecomment-1197895181). If you want this field to be documented, you likely need to uncomment it with a default value

Reply:
>Commenting this out will mean it's undocumented in the Helm reference

Yes, that's intentional.

Reply:
Can we set it with `terminatePodConnections: ~` instead? That _should_ provide no overriding default but allow the docs generation to work.